uplink/metainfo/kvmetainfo/objects_test.go

```diff
@@ -391,234 +391,6 @@ func TestListObjects(t *testing.T) {
-				result:  []string{"xa", "xaa", "xb", "xbb", "xc"},
-			}, {
-				options: options("a/", "xb", storj.Backward, 0),
-				result:  []string{"xa", "xaa", "xb"},
-			}, {
-				options: optionsRecursive("", "b/yaa", storj.Backward, 5),
-				more:    true,
-				result:  []string{"a/xc", "aa", "b", "b/ya", "b/yaa"},
-			}, {
-				options: options("a/", "xbb", storj.Backward, 2),
-				more:    true,
-				result:  []string{"xb", "xbb"},
-			}, {
-				options: options("", "", storj.Before, 0),
-				result:  []string{"a", "a/", "aa", "b", "b/", "bb", "c"},
-			}, {
-				options: options("", "`", storj.Before, 0),
-				result:  []string{},
-			}, {
-				options: options("", "a", storj.Before, 0),
-				result:  []string{},
-			}, {
-				options: options("", "b", storj.Before, 0),
-				result:  []string{"a", "a/", "aa"},
-			}, {
-				options: options("", "c", storj.Before, 0),
-				result:  []string{"a", "a/", "aa", "b", "b/", "bb"},
-			}, {
-				options: options("", "ca", storj.Before, 0),
-				result:  []string{"a", "a/", "aa", "b", "b/", "bb", "c"},
-			}, {
-				options: options("", "", storj.Before, 1),
-				more:    true,
-				result:  []string{"c"},
-			}, {
-				options: options("", "`", storj.Before, 1),
-				result:  []string{},
-			}, {
-				options: options("", "a/", storj.Before, 1),
-				result:  []string{"a"},
-			}, {
-				options: options("", "c", storj.Before, 1),
-				more:    true,
-				result:  []string{"bb"},
-			}, {
-				options: options("", "ca", storj.Before, 1),
-				more:    true,
-				result:  []string{"c"},
-			}, {
-				options: options("", "", storj.Before, 2),
-				more:    true,
-				result:  []string{"bb", "c"},
-			}, {
-				options: options("", "`", storj.Before, 2),
-				result:  []string{},
-			}, {
-				options: options("", "a/", storj.Before, 2),
-				result:  []string{"a"},
-			}, {
-				options: options("", "bb", storj.Before, 2),
-				more:    true,
-				result:  []string{"b", "b/"},
-			}, {
-				options: options("", "c", storj.Before, 2),
-				more:    true,
-				result:  []string{"b/", "bb"},
-			}, {
-				options: options("", "ca", storj.Before, 2),
-				more:    true,
-				result:  []string{"bb", "c"},
-			}, {
-				options: optionsRecursive("", "", storj.Before, 0),
-				result:  []string{"a", "a/xa", "a/xaa", "a/xb", "a/xbb", "a/xc", "aa", "b", "b/ya", "b/yaa", "b/yb", "b/ybb", "b/yc", "bb", "c"},
-			}, {
-				options: options("a", "", storj.Before, 0),
-				result:  []string{"xa", "xaa", "xb", "xbb", "xc"},
-			}, {
-				options: options("a/", "", storj.Before, 0),
-				result:  []string{"xa", "xaa", "xb", "xbb", "xc"},
-			}, {
-				options: options("a/", "xb", storj.Before, 0),
-				result:  []string{"xa", "xaa"},
-			}, {
-				options: optionsRecursive("", "b/yaa", storj.Before, 5),
-				more:    true,
-				result:  []string{"a/xbb", "a/xc", "aa", "b", "b/ya"},
-			}, {
-				options: options("a/", "xbb", storj.Before, 2),
-				more:    true,
-				result:  []string{"xaa", "xb"},
```

Comment:
Why are these test cases removed?

Reply:
At the moment we don't support `direction` for objects listing with new metainfo requests.

Reply:
Will the next PRs add such support? In such, wouldn't it be better to comment these tests instead of removing them?

Reply:
There are no plans for such PR but I'm ok with leaving this tests commented until we will be sure about `direction` future.